README.md

```diff
@@ -22,6 +22,11 @@ $ bin/evm2wasm.js -e `evm_bytecode_file` -o `wasm_output_file`
 $ bin/evm2wasm.js -e `evm_bytecode_file` -o `wasm_output_file` --wast
 ```
 
+#### Transcompile EVM to WAST with gas metering per transpiled EVM opcode (not per branch segment)
+```
+$ bin/evm2wasm.js -e `evm_bytecode_file` -o `wasm_output_file` --wast --chargePerOp
```

Comment:
I think I'd prefer something like `--charge-per-op` on the CLI.

Reply:
fair enough